modules/spark/src/test/scala/org/apache/ignite/spark/IgniteCatalogSpec.scala

```diff
@@ -39,9 +39,9 @@ class IgniteCatalogSpec extends AbstractDataFrameSpec {
         it("Should observe all available SQL tables") {
             val tables = igniteSession.catalog.listTables.collect()
 
-            tables.length should equal (3)
+            tables.length should equal(6)
 
-            tables.map(_.name).sorted should equal (Array("CITY", "EMPLOYEE", "PERSON"))
+            tables.map(_.name).sorted should equal(Array("CITY", "EMPLOYEE", "PERSON", "PUBLIC.CITY", "PUBLIC.PERSON", "cache3.EMPLOYEE"))
```

Comment:
CITY and PERSON tables are present in the PUBLIC schema because they were created with the CREATE TABLE syntax which [according to the docs](https://apacheignite-sql.readme.io/docs/create-table) always uses the public schema, whereas the EMPLOYEE table is present in the cache3 schema because it was created as a cache using CacheConfiguration and so resides in the schema defaulting to the cache name. I'll add a comment as such in the test.